brunch-config.js

```diff
@@ -160,9 +160,19 @@ exports.config = {
     assetsmanager: {
         copyTo: {
           '/templates/' : [ 'src/app/templates/*.tmpl' ],
+          '/cms/api/' : [ 'src/cms/api/*.php' ],
           '/cms/images/previews/' : [ 'src/app/templates/previews/*.png' ],
         }
     },
+    // after compile, run any shell commands (or `node my_node_cmd`)
+    afterBrunch: [
+      'mkdir -p www/downloads',
+      'chmod 777 www/downloads',
+      'chmod 777 www/demo',
+      'chmod 777 www/demo/images',
+      // move the example index page out of assets, to the web root
```

Comment:
assets get copied "as is" ... so we must manually move the index.php we keep in assets to the webroot after build ..